<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-3V017FK" failures="1" tests="1" name="Test" time="35.688" errors="0" timestamp="2023-07-01T11:44:05 IST">
  <testcase classname="test.PB_TC_1020_ValidatedPolicyUserName1" name="validatedusername" time="3.139">
    <failure type="java.lang.AssertionError" message="tc is failed actual and exp value are not maching expected [Kamini Harde1] but found [Kamini Harde]">
      <![CDATA[java.lang.AssertionError: tc is failed actual and exp value are not maching expected [Kamini Harde1] but found [Kamini Harde]
at test.PB_TC_1020_ValidatedPolicyUserName1.validatedusername(PB_TC_1020_ValidatedPolicyUserName1.java:80)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- validatedusername -->
</testsuite> <!-- Test -->
